test-cases/gutenberg/writing-flow/multiline-components.md

```diff
@@ -12,6 +12,7 @@ Test the next steps on:
 
 **Known Issues**
 - On captions, pressing Enter at the end of a caption splits the block, but pressing enter in the middle of a caption's text does nothing. Discussed in [this comment](https://github.com/WordPress/gutenberg/pull/22928#issuecomment-640879690).
+- On Android, tapping Enter to split a quote or pullquote citation both splits the block and adds a newline to the citation ([#2498](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2498))
```

Comment:
> I just noticed I hadn't hit "Submit review" on this yesterday!

I've definitely done that before! 😂

> Would it be worth noting here that this doesn't happen all the time, and depends on the keyboard used? I've added a comment to the ticket you recently created, but it might be worth clarifying here, wdyt?

I feel like it might be ok to leave that out since it would just mean that everything would work fine as far as the test case was concerned. I think it is best if we can keep the descriptions here as short as possible and basically they should just serve as an alert to someone running the tests that there is an issue here and if they want to know more about it then they should go look at the actual issue. I only say that because I don't think we'll ever be able to keep the test case known issues up-to-date with the latest findings on the issue ticket.

With all that said, I certainly don't think it would do any harm to add some more info here. Just let me know if you still think it would help some more, and I would be happy to add it.

Reply:
I agree we can leave it out 👍